docs/arcaflow/workflows/expressions.md

```diff
@@ -14,44 +13,60 @@ This page explains the language elements of expressions.
 
 ## Literals
 
-Literals are a very important part of the language. It's required to access any value that's known.
+Literals represent values described in an expression, as opposed to values referenced from other sources.
 
-### string
+### String
 
 string literals start and end with a matched pair of either single quotes `'` or double quotes `"`, and have zero or more characters between the quotes.
 
-Strings may have escaped values. The most important ones are for backslashes (`\\` for `\`), or for new lines `\n`.
-
-For example, to have this in a string:
+Strings may have escaped values. The most important ones are for backslashes (`\\` for `\`) or for embedded newlines `\n`.
```

Comment:
> Of particular note is that YAML already provides for [escape sequences](https://yaml.org/spec/1.2.2/#escaped-characters) in [scalar values](https://yaml.org/spec/1.2.2/#scalars), so, unless the user works really hard, I suspect that ARCA's code will never see an escape sequence (except from tests...).

Now that I think about it, if the user wants to define a reference in his YAML file for a value which is a string containing a backslash, I think he's in for a significant challenge.  That is, unless I'm confused, he'll need to escape the backslash once to get it past the YAML parser, and then he'll need to escape it **_again_** to get it past ARCA's evaluation.  (I.e., it'll need to be `"\\\\"` so that exactly one of them wins through.)

That is, the user needs to know that there are exactly _two_ escape evaluation passes between his YAML file and the consumption of the value.

And, that leads to the question of _why_ is ARCA replacing escapes if YAML is already doing it?

And, I have another question:  while the escapes that we're talking about here are in values coming from the YAML file, what about inputs coming from steps?  Do those undergo escape replacement?  If so, who is responsible for _inserting_ the escapes so that the references to the output end up getting the appropriate values?